nemo/collections/nlp/data/language_modeling/megatron/dataset_utils.py

```diff
@@ -634,6 +610,7 @@ def build_dataset(index, name):
             )
 
             if dataset_type == DSET_TYPE_ICT:
+                raise NotImplementedError("ICT dataset is not implemented yet.")
                 dataset = ICTDataset(
                     block_dataset=indexed_dataset,
                     title_dataset=title_dataset,
```

Comment:
Can you comment the instantiation below?

Reply:
Done!